tests/preprocessing/test_positionfixes.py

```diff
@@ -1,116 +1,225 @@
 import datetime
 import os
 import sys
+from math import radians
 
 import geopandas as gpd
 import numpy as np
 import pandas as pd
-from sklearn.cluster import DBSCAN
+import pytest
 from shapely.geometry import Point
-from math import radians
+from sklearn.cluster import DBSCAN
 
 import trackintel as ti
 from trackintel.geogr.distances import haversine_dist
 
+
+@pytest.fixture
+def pfs_geolife():
```

Comment:
should these `fixture` functions also be placed at the end of the file (following convention)? And their name with prefix `_`?

Reply:
I think for fixtures the `_` convention doesn't really apply because you wouldn't import anything from a test script anyways. I tried to follow the [pytest documentation for fixtures](https://docs.pytest.org/en/stable/fixture.html) and there the fixtures are at the top. I think the idea is to organize the file similar to the life cycle of a test (Arrange, Act, Assert, Cleanup)